tap/passive_tapper.go

```diff
@@ -187,7 +189,7 @@ func initializePassiveTapper(opts *TapOpts, outputItems chan *api.OutputChannelI
 	diagnose.InitializeErrorsMap(*debug, *verbose, *quiet)
 	diagnose.InitializeTapperInternalStats()
 
-	mainPacketInputChan = make(chan source.TcpPacketInfo)
+	mainPacketInputChan = make(chan source.TcpPacketInfo, 1000)
```

Comment:
In `gopacket` they use `1000` as the channel buffer size in here:

https://github.com/google/gopacket/blob/v1.1.19/packet.go#L860

So, I guess `1000` is OK?